substra/sdk/backends/remote/backend.py

```diff
@@ -64,9 +64,9 @@ def get_performances(self, key):
             performances.compute_plan_metadata.append(compute_plan.metadata)
 
             performances.worker.append(test_task["compute_task"]["worker"])
-            performances.testtask_key.append(test_task["compute_task"]["key"])
-            performances.metric_name.append(test_task["metric"]["name"])
-            performances.testtask_rank.append(test_task["compute_task"]["rank"])
+            performances.task_key.append(test_task["compute_task"]["key"])
+            performances.function_name.append(test_task["metric"]["name"])
```

Comment:
Did you create a card to delete "metric" ref on the stack ?

Reply:
Yep should be in "ready" column :) I didn't know the exhaustive list of repos where we still have mentions of metric,  feel free to complete it 👍